SK.WebApp/src/services/consts.ts

```diff
@@ -0,0 +1,11 @@
+const  Culture = {
+    POLISH: "pl-PL",
+    ENGLISH: "en-US",
+};
```

Comment:
It's reaaaally good that you define constants that could be reused. This directory structure needs to go, though. 

When you define something as `service` and `consts`, and when you combine these two (if you'd used `helpers` you'd have made a combo) it shows that you shy away from actually naming things.

Here's my proposal. `Culture` could be set up like this:

```ts
// SK.WebApp/src/language.ts
export enum language {
  english = 'english',
  polish = 'polish',
}

export enum languageCode {
  en_US = 'en_US',
  pl_PL = 'pl_PL',
}

export type anyLanguage = keyof typeof language;
export type anyLanguageCode = keyof typeof languageCode;

export const languageToLanguageCodes: Record<
  anyLanguage,
  anyLanguageCode[]
> = {
  [language.english]: ['en-US'],
  [language.polish]: ['pl-PL'],
};
```

Why that structure? It's strictly typed, so it's easier to maintain and less error-prone.
Why multiple language codes instead of just one? Browsers define language in [`BCP47`](https://tools.ietf.org/rfc/bcp/bcp47.txt). One language could have multiple language codes. In our case `en-US` and `en-GB` are the same but using list instead of single string is future-proof.